src/test/web/customReporter.ts

```diff
@@ -66,7 +95,7 @@ function writeReportProgress(message: Message) {
         messages.push(message);
         if (message.event === constants.EVENT_RUN_END) {
             const ext = extensions.getExtension(JVSC_EXTENSION_ID_FOR_TESTS)!.extensionUri;
-            const logFile = Uri.joinPath(ext, 'webtest.json');
+            const logFile = Uri.joinPath(ext, 'testresults.json');
```

Comment:
new name, all tests will now generate this file (not just web tests).
I.e. even regular (desktop) tests will generate this file, so we can dump the test results in a separate job as we did in web test, thus producing a cleaner, simpler  & consistent output for all tests.